msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/ConfidentialClientApplication.java

```diff
@@ -208,6 +215,12 @@ public ConfidentialClientApplication.Builder appTokenProvider(Function<AppTokenP
             throw new NullPointerException("appTokenProvider is null") ;
         }
 
+        public ConfidentialClientApplication.Builder useSharedCache(boolean val) {
```

Comment:
The user experience in MSAL.net is enable this through cache options. Are we not planning to be consistent with it?